### ISSUE TITLE
Questions on hyperparameters and test metrics

### ISSUE THREAD

**Issue by ZixunWang** [JunwenBai/c-gmvae#4]
Hi Junwen,

In Appendix B.1, it says `learning rate`, `dropout ratio` and `weight decay` are tuned by grid search. It is highly appreciated if you could kindly share the final selected hyperparameters to help us reproduce results. Thx.

And I find in `test.py` you use different thresholds to select best metrics. I think it might be strange to do so.

**Comment by JunwenBai** [comment]
For fair comparison with baselines, in testing, we adopted the same procedure as LaMP (https://github.com/QData/LaMP). Feel free to check out their `README` for explanation. You can also use 0.5 for all compared methods to have new numbers for comparison.
